syntax_and_semantics/constants.md

```diff
@@ -28,3 +28,39 @@ end
 
 TEN #=> 10
 ```
+
+# Psuedo Constants
```

Comment:
https://en.m.wiktionary.org/wiki/psuedo ☺

Reply:
hah, good catch

Reply:
I tried to think that I'd typo'd that. "You know how to spell," I told myself. 

And then I found all 6 times I'd typed that. 😅